Bugsnag/Assets/Bugsnag/Editor/BuildPreprocessor.cs

```diff
@@ -9,16 +9,22 @@ public class BuildPreprocessor : IPreprocessBuildWithReport
 
     public void OnPreprocessBuild(BuildReport report)
     {
+        //This will be enabled once IL2CPP source mapping is supported
```

Comment:
I'm not sure I understand the comment - it seems to be enabled now?

Reply:
was a mistake